lib/game.rb

```diff
@@ -99,18 +100,10 @@ def attempt_fire_on_player_ship
     until !@player_board.cells[coord].fired_upon?
      coord = valid_range.shift
     end
-    puts "Now it's my turn!"
     @player_board.cells[coord].fire_upon
-    sleep 2
     render_playspace
-    case feedback(@player_board, coord)
-    when :hit
-      p "I hit one of your ships!"
-    when :missed
-      p "I Missed!"
-    when :sunk
-      p "I Sunk your #{@player_board.cells[coord].ship.name}!"
-    end
+    sleep(0.5)
+    puts "I #{feedback(@player_board, coord).to_s} your ship"
```

Comment:
Nice choice to be less specific until we have some solid method to handle messaging.